Update end user README with Demo instructions (#394)
eclipse-pass/main: docs/user/README.md

# Eclipse PASS Demo Instructions

> 1. Login: `nih-user` // `moo` 

Reviewer: ```suggestion
1. Login with username: `nih-user` and password: `moo` 
```

2. Enter submission workflow by clicking on the "Start new submission" button
3. Enter DOI `10.1039/c7an01256j` into the DOI input field

> 4. DOI service should retrieve data and populate submission 'title' and 'journal' after it returns

Reviewer: ```suggestion
4. PASS will retrieve data and populate the submission 'title' and 'journal' fields. You may need to wait a moment for this to occur.
```

5. Grants step: go to next step and select an NIH grant: Click anywhere on the row to select grant: R07EY012124 - Regulation of Synaptic Plasticity in Visual Cortex
  - When a grant is selected, it should appear in a separate “Grants added to submission” table, appearing above the “Available grants” table
7. Policies step: accept the currently selected policies, click Next to move on
  - Default selection for NIH policy is expected in most cases
8. Repositories step: accept the selected repositories, click Next to move on

>   - J10P should be selected by default

Reviewer: ```suggestion
  - JScholarship should be selected by default
```

7. Metadata (details) step: accept the pre-filled metadata, click Next to move on
  - Metadata form should have most fields pre-populated with data from the DOI with each such field being read-only

> 8. Files step: add any file to the submission), click Next to move on

Reviewer: ```suggestion
8. Files step: add any file to the submission, click Next to move on.
```
